lrmd/lrmd_private.h

```diff
@@ -33,6 +34,8 @@
 #define LRMD_ISOLATION_PROVIDER ".isolation"
 
 GHashTable *rsc_list;
+regex_t *version_format_regex;
+GHashTable *ra_version_hash;
```

Comment:
These should be declared "extern" here. I'm not sure why rsc_list isn't; it should be.